tooling/reproducible/linux_repro_build_compare.sh

```diff
@@ -61,9 +62,13 @@ setEnvironment() {
 }
 
 cleanBuildInfo() {
+  # shellcheck disable=SC3043
+  local DIR="$1"
   # BUILD_INFO name of OS level build was built on will likely differ
-  sed -i '/^BUILD_INFO=.*$/d' "jdk-${TEMURIN_VERSION}/release"
-  sed -i '/^BUILD_INFO=.*$/d' "compare.$$/jdk-${TEMURIN_VERSION}/release"
+  sed -i '/^BUILD_SOURCE=.*$/d' "${DIR}/release"
```

Comment:
I'm now thinking this is not right, we should be using the identical BUILD_SOURCE, ie the exact same temurin-build scripts version...?

Reply:
right, I see now, so yes this is being build by linux_repro_build_compare.sh, not the build_farm which adds BUILD_SOURCE , BUILD_REPO, BUILD_SOURCE_REPO

Reply:
Just to confirm so should I remove those three lines or keep as is?

Reply:
@andrew-m-leonard